gradle/stringGenerator.gradle

```diff
@@ -18,7 +19,11 @@ private def generate(Map<String, String> sources) {
         printDiffs(diffs)
         throw new IllegalStateException("Strings source can't be different")
     }
-    def defaultLang = getDefaultLangKey(jsonMap.keySet())
+    def defaultLang = userDefaultLanguage
```

Comment:
if (defaultLang != null && !jsonMap.keySet().contains(defaultLang)) {
    throw new Exception...
} 
I think need check like that